src/drivers/gitlab.js

```diff
@@ -9,7 +9,7 @@ const ProxyAgent = require('proxy-agent');
 
 const { fetchUploadData, download, exec } = require('../utils');
 
-const { IN_DOCKER, CI_PIPELINE_ID } = process.env;
+const { IN_DOCKER, CI_PIPELINE_ID, CML_GL_DOCKER_VOLUMES } = process.env;
```

Comment:
Wouldn't it be better to add it as a [hidden] command-line option instead of accessing `process.env` from many places?

https://github.com/iterative/cml/blob/53e1547abfc349b8e75ee71f02709b331db2e59c/bin/cml/runner.js#L413

Reply:
Yes, i said this in slack, this way is to have a fast access to test. I had this code long time ago. But definitely we must expose the param (can be visible) and prepare the equivalent pr in tpi

Reply:
Yes and don't think we need GL/gitlab in the CLI option's name. We can raise a "warning: ignored" if it's specified with an unsupported provider.

Reply:
We are on the same page